test/services/user-group/services/classSuccessor.test.js

```diff
@@ -191,7 +191,7 @@ describe('classSuccessor service', () => {
 			throw new Error('should have failed');
 		} catch (error) {
 			expect(error.message).to.not.equal('should have failed');
-			expect(error.message).to.equal("You don't have the permission USERGROUP_CREATE.");
+			expect(error.message).to.equal("You don't have one of the permissions: USERGROUP_CREATE.");
```

Comment:
Something is missing?

Reply:
it's just a new error message because the hasPermission method now also supports multiple permissions

Reply:
I expect that throw new Forbidden(`You don't have one of the permissions: ${permissionNames.join(', ')}.` will return other output as USERGROUP_CREATE.